loops-and-arrays.js

```diff
@@ -41,6 +51,10 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i] + " is " + squares[i] * squares[i] * squares[i]);
+}
+
```

Comment:
Vervolgens moet de huidige waarde in de array overschreven worden met: "Het volume van [x] is [y]"
ook hier weer wordt gevraagd om de waarde van de arrey te overschrijven. als je bij jouw na de for loop nu console.log(squares) aanroept geeft hij nog steeds de oude waarde en niet de nieuwe. je mag in javascript in tegenstelling tot java(backend) zowel tekst als getallen opslaan in de zelfde array en dus kan je hem overschrijven met  de zin "Het volume van " + squares[i] + " is " + squares[i] * squares[i] * squares[i]

Reply:
Yes nu begrijp ik het inderdaad!